homeassistant/components/sensor/youless.py

```diff
@@ -0,0 +1,119 @@
+"""
+@ Author      : Gerben Jongerius
+@ Date        : 04/29/2018
+@ Description : Youless Sensor - Monitor power consumption. This component will add the following sensors
+                   - Current power consumption (in W)
+                   - Current tick count, since the Youless meter started running (in W)
+"""
```

Comment:
Please use the same style for the file header as the other files do.

Reply:
I have changed the header based on one I found in the sensor directory